test/pkg/gitea/test.go

```diff
@@ -227,3 +228,29 @@ func WaitForPullRequestCommentMatch(ctx context.Context, t *testing.T, topts *Te
 		i++
 	}
 }
+
+func CheckIfPipelineRunsCancelled(t *testing.T, topts *TestOpts) {
+	i := 0
+	for {
+		list, err := topts.Clients.Clients.Tekton.TektonV1beta1().PipelineRuns(topts.TargetNS).
+			List(context.Background(), metav1.ListOptions{
+				LabelSelector: fmt.Sprintf("app.kubernetes.io/managed-by=%v", pipelinesascode.GroupName),
```

Comment:
can you check the repo name directly? instead of pipelinerun? i'd like one day to be able to run all those tests in a single namespace

Reply:
updated.
cancelled e2e.. rate limited last time
will run after sometime